### PR TITLE
Get rid of BridgeSupportTest hierarchy 

### PR DESCRIPTION
**Pull request by nathanieliov**
Get rid of the BridgeSupportTest hierarchy, move those methods to util classes, and make test classes independent.

### CODE REVIEW

**Review by jeremy-then — APPROVED** [review]
LGTM.

File: rskj-core/src/test/java/co/rsk/peg/utils/BridgeStorageProviderUtil.java [hunk 1]
```diff
@@ -1,23 +1,14 @@
-package co.rsk.peg;
+package co.rsk.peg.utils;
```
[the remark this answers is not in the document]
Comment (nathanieliov) [reply rskj-core/src/test/java/co/rsk/peg/utils/BridgeStorageProviderUtil.java:1]: I'll get rid of that class by moving its methods into the test class that uses them.

File: rskj-core/src/test/java/co/rsk/peg/utils/BridgeStorageProviderUtil.java [hunk 2]
```diff
@@ -27,28 +18,12 @@
...
-    }
-
-    protected static Repository createRepository() {
+public final class BridgeStorageProviderUtil {
+    public static Repository createRepository() {
         return new MutableRepository(new MutableTrieCache(new MutableTrieImpl(null, new Trie())));
     }
 
-    protected static void mockChainOfStoredBlocks(BtcBlockStoreWithCache btcBlockStore, BtcBlock targetHeader, int headHeight, int targetHeight) throws BlockStoreException {
+    public static void mockChainOfStoredBlocks(BtcBlockStoreWithCache btcBlockStore, BtcBlock targetHeader, int headHeight, int targetHeight) throws BlockStoreException {
```
[the remark this answers is not in the document]
Comment (nathanieliov) [reply rskj-core/src/test/java/co/rsk/peg/utils/BridgeStorageProviderUtil.java:26]: Agree.

**Review by marcos-iov — APPROVED** [review]
This is gonna need some rebase now

**Pull request merged**